### ISSUE TITLE
Inconsistent Required status for First name and Last name fields in Profiles and Users classes in Admin Console

### ISSUE THREAD

**Issue by werberger** [CiviWiki/OpenCiviWiki#1465]
### Description

In the Admin Console, you are not able to save a _Profile_ with empty _First name_ or _Last name_.

However, you are able to save a _User_ with name fields empty.

### Further details

If the desired outcome is that the _Profile_ can be saved with empty name fields then I think an easy fix may be:

`class Profile(models.Model):`
`    user = models.OneToOneField(User, on_delete=models.CASCADE, related_name="profile")`
`    first_name = models.CharField(max_length=63, blank=**False**)` change to **True**
`    last_name = models.CharField(max_length=63, blank=**False**)` change to **True**

Or if "Anonymous" is the desired default (as I've seen mentioned), a possibly a partial solution would be to add `default="Anonymous"` for _first_name_ and change _blank_ to **True** as above. However, this would still mean that _first_name_ would be required, of course.

**Comment by ilaiyengar** [comment]
Hi, I am a first time contributor! Would this be a good first issue that I can assist with?

**Comment by brylie** [comment]
@werberger may want to claim this issue.

**Comment by werberger** [comment]
yeah. thanks @brylie and sorry @ilaiyengar - if it's any consolation it will also be my first contribution.

**Comment by Nehemiah60** [comment]
Hello, @brylie. Kindly assign me this

**Comment by LuckyKhoza-crypto** [comment]
It's still open, please assign it to me.

**Comment by stlynnxx** [comment]
Hi, if this is still open, could I please be assigned?
